### DISCUSSION TITLE
"Welcome to Cypress!" message comes up every time I run cypress:open

### DISCUSSION THREAD

**Discussion by richardszanyi-kasa** [cypress-io/cypress#20780] [Questions and Help]
Hello all, 

So I fiddled around with restructuring our currently existing solution regarding folders in Cypress. 
After that, every time I run cypress open, and opens up the dashboard, this message comes up:
![image](https://user-images.githubusercontent.com/101114202/160113115-0a8dc220-97f4-46db-8323-c4ccc5651c18.png)

Does anyone know a quick solution to that matter? 

Thanks!

**Comment by pmillspaugh** [comment]
I am facing the same issue. I moved my `cypress/` folder and `cypress.config.ts` file from the root of my project into an `src/` folder, which I suspect has something to do with it. 

Each time I run `cypress open --project src/cypress`, I see the welcome screen:
<img width="765" alt="Screen Shot 2022-07-07 at 8 13 01 PM" src="https://user-images.githubusercontent.com/73900714/177891532-bc0d69c6-1feb-49de-8611-e4b85643a787.png">

Followed by the choose browser screen:
<img width="569" alt="Screen Shot 2022-07-07 at 8 13 37 PM" src="https://user-images.githubusercontent.com/73900714/177891586-cc98b573-5d6f-4097-846c-72c11f1577a0.png">

I would love to avoid clicking through both each time I run tests. Is there a config option I can add to do so? Thanks in advance!

**Reply by danfooks** [reply]
@pmillspaugh I also receive the welcome screen every time I run Cypress with `npx cypress open --config-file cypress.config.js`. This is inconvenient, as I already know I want to access my e2e tests with a specific browser.

**Reply by emilyrohrbough** [reply]
@pmillspaugh If you know what testing type and/or browser you'd like to run you can use one of the following:
- `cypress open --project src/cypress --e2e`
- `cypress open --project src/cypress --e2e --browser=chrome`
- `cypress open --project src/cypress --component`
- `cypress open --project src/cypress --component --browser=firefox`

**Reply by pmillspaugh** [reply]
Works perfectly — thank you so much, @emilyrohrbough!

**Comment by davifsroberto** [comment]
I configured this command:

`"cy:open:chrome": "cypress open --e2e --browser chrome",
`

And then you decide which browser and which type of test.
